src/xarray_regrid/methods/conservative.py

```diff
@@ -72,8 +72,10 @@ def conservative_regrid(
 
     # Make sure the regridding coordinates are sorted
     coord_names = [coord for coord in target_ds.coords if coord in data.coords]
-    target_ds_sorted = target_ds.sortby(coord_names)
-    data = data.sortby(list(coord_names))
+    target_ds_sorted = target_ds.copy()
```

Comment:
```suggestion
    target_ds_sorted = xr.Dataset(target_ds.coords)
```
We don't need the values of the target dataset, only the coordinates. This could reduce any potential copies of the target_ds data.